src/rust/engine/src/externs/mod.rs

```diff
@@ -255,6 +255,21 @@ pub fn val_to_log_level(obj: &PyObject) -> Result<log::Level, String> {
   res.map(|py_level| py_level.into())
 }
 
+/// Link to the Pants docs using the current version of Pants.
+pub fn docs_url(slug: &str) -> String {
+  let gil = Python::acquire_gil();
+  let py = gil.python();
+  let docutil = py.import("pants.util.docutil").unwrap();
+  py.eval(
```

Comment:
Why use `eval` here? After importing `cpython::ObjectProtocol`, you should just be able to do:

```
docutil.call_method(py, "docs_url", (&slug,), None)` 
```

And if that wasn't work, do:

```
let docs_url = docutil.getattr(py, "docs_url").unwrap();
docs_url.call(py, (&slug,), None) ...
```

Reply:
See https://docs.rs/cpython/0.5.2/cpython/trait.ObjectProtocol.html#method.call

I may have gotten the need for `&slug` vs `slug` wrong but this form should work.

Reply:
Please let me know if it doesn't work. I prefer we avoid having to reason about the security aspects of using `eval`.

Reply:
`PyModule.call()`. That's super cool! Thanks Tom!